### PR TITLE
Feature/refactor session logic

### PR DESCRIPTION
**Pull request by rolandvarga**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,27 +3,73 @@ A maintenance scheduler UI for Prometheus AlertManager
...
+
+
+Use -h flag to list available options.
+
 ### Configuration & Running
 The tool relies on a YAML config file to specify the Alertmanager address it is supposed to send requests to:
 ```yaml
 ---
-alertmanager_api: "http://localhost:9093/api/v2"
+alertmanager_api: "http://localhost:9093/"
```
Comment (louisfelix) [inline README.md]:
```suggestion
alertmanager_url: "http://localhost:9093/"
```

File: README.md [hunk 2]
```diff
@@ -3,27 +3,73 @@ A maintenance scheduler UI for Prometheus AlertManager
...
+
+```bash
+./alertmanager-maintenance-scheduler --config.file=/path/to/config.yml
+```
+
+The exporter's Alertmanager API connection can also be configured by defining the following environment variable(s). If they are present, they will take precedence over the corresponding variables in the config file.
+
+Environment Variable | Description
+---------------------| -----------
+ALERTMANAGER_URL | URL of the exported alertmanager api (eg: "http://localhost:9093/api/v2")
```
Comment (louisfelix) [inline README.md]:
```suggestion
ALERTMANAGER_URL | URL of the alertmanager (eg: "http://localhost:9093/")
```

File: README.md [hunk 3]
```diff
@@ -3,27 +3,73 @@ A maintenance scheduler UI for Prometheus AlertManager
...
+```
+
+## Configuration
+
+An example can be found in
+[sample-config.yml](https://github.com/FXinnovation/alertmanager-maintenance-scheduler/blob/master/sample-config.yml).
+
+Configuration element | Description
+--------------------- | -----------
+alertmanager_url | (Mandatory) URL of the exported alertmanager api (eg: "http://localhost:9093/")
```
Comment (louisfelix) [inline README.md]: Is it exported? Or just "URL of the alertmanager"

File: README.md (hunk 1 above)
Comment (rolandvarga) [reply README.md]: changed

File: README.md (hunk 2 above)
Comment (rolandvarga) [reply README.md]: changed

File: README.md (hunk 3 above)
Comment (rolandvarga) [reply README.md]: changed

**Pull request merged**